### DISCUSSION TITLE
How can I get access to copilot chat?

### DISCUSSION THREAD

**Discussion by mdevadass** [community/community#57862] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

 I am currently investigating the potential enhancement of developer productivity through the utilization of Copilot chat. I have been patiently waiting on the Copilot Chat waitlist for the past three weeks. Is there any way to accelerate the process and gain access to Copilot chat sooner?

**Comment by plon** [comment, marked as answer]
It is not possible to accelerate the process. I waited a few months to gain access to Copilot Chat.
